recipes/qt/6.x.x/test_package/meson.build

```diff
@@ -1,6 +1,6 @@
 project('test_package', 'cpp')
-qt5 = import('qt5')
-qt5_dep = dependency('qt5', modules: ['Core'])
-moc_files = qt5.preprocess(moc_headers : 'greeter.h')
+qt6 = import('qt6')
+qt6_dep = dependency('qt6', modules: ['Core'])
+moc_files = qt6.preprocess(moc_headers : 'greeter.h', qresources : 'example.qrc')
```

Comment:
if meson is properly supported (at least meson 0.57.0 I think), then you should also remove `False and` in the definition of ` def _meson_supported` in `test_package/conanfile.py`

Reply:
I'm not sure about meson. I guess for a different PR :-).